libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/route/NavigationRouteEx.kt

```diff
@@ -0,0 +1,52 @@
+package com.mapbox.navigation.base.internal.route
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.LegAnnotation
+import com.mapbox.navigation.base.internal.SDKRouteParser
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigation.base.route.toNavigationRoute
+import com.mapbox.navigator.Navigator
+import com.mapbox.navigator.RouteInterface
+
+/**
+ * Internal handle for the route's native peer.
+ */
+fun NavigationRoute.nativeRoute(): RouteInterface = this.nativeRoute
```

Comment:
RouteInterface is almost the same Route**r**Interface, in the first iteration I cannot get why you need an instance of NN Router in NavigationRoute 😄 . Could you add internal type alias? Would be great to set for interface another name in NN cc @mskurydin

Reply:
+ 1
Names sound too similar.

Reply:
I'll let Nav Native refactor if needed.